onset_maproom/environment.yml

```diff
@@ -19,6 +19,7 @@ dependencies:
   - xarray
   - yaml
   - zarr
+  - bottleneck
```

Comment:
Please update environment_linux.yml as follows (on a linux machine):
- conda deactivate
- conda env remove -n enactsmaproom
- conda env create -f environment.yml -n enactsmaproom
- conda activate enactsmaproom
- conda env export > environment_linux.yml

Then test the application thoroughly, because in addition to adding bottleneck this will update everything to the latest version, and it's possible that one of our dependencies released a breaking change since the last time we rebuilt the environment.

(I'm not bothering to document this somewhere more permanent because I'm working on what I hope will be a better way of managing environments.)